packages/lesswrong/components/tagging/subforums/TagSubforumPage2.tsx

```diff
@@ -185,17 +186,33 @@ const TagSubforumPage2 = ({classes}: {
     skip: !query.flagId
   })
 
-  const { results: userTagRelResults } = useMulti({
+  const skipUserTagRel = !tag || !currentUser
+
+  const { results: userTagRelResults, loading: userTagRelLoading } = useMulti({
     terms: { view: "single", tagId: tag?._id, userId: currentUser?._id },
     collectionName: "UserTagRels",
     fragmentName: "UserTagRelDetails",
-    // Create a new UserTagRel if none exists. The check for the existence of tagId and userId is
-    // in principle redundant because of `skip`, but it would be bad to create a UserTagRel with
-    // a null tagId or userId so be extra careful.
-    createIfMissing: tag?._id && currentUser?._id ? { tagId: tag?._id, userId: currentUser?._id } : undefined,
-    skip: !tag || !currentUser
+    skip: skipUserTagRel
   });
-  const userTagRel = userTagRelResults?.[0];
+
+  const { create: createUserTagRel, data: createdUserTagRelData, called: createUserTagRelCalled } = useCreate({
+    collectionName: "UserTagRels",
+    fragmentName: "UserTagRelDetails",
+  })
+
+  useEffect(() => {
```

Comment:
This refactoring might be relevant to the EA forum if subforums are still around; I think it should function the same way as the previous `createIfMissing` behavior (if perhaps a bit slower) but can't trivially test it.